src/sdk.ts

```diff
@@ -185,14 +267,12 @@ class APWineSDK {
    * Updates the signer on an existing APWineSDK instance.
    * @param signer - A transaction signer.
    */
-  updateSigner(signer: Signer) {
+  updateSigner(signer: Signer, use: boolean = true) {
```

Comment:
can we change 'use' to something more meaningful?

Reply:
done